.gitignore

```diff
@@ -0,0 +1,28 @@
+# Xcode
```

Comment:
I usually use these standard ones: [macOS](https://github.com/github/gitignore/blob/master/Global/macOS.gitignore) and [Xcode](https://github.com/github/gitignore/blob/master/Global/Xcode.gitignore) but maybe it's too much for this, there is no many things to ignore